determiner.py

```diff
@@ -0,0 +1,115 @@
+#! /usr/bin/env python3
+
+import argparse
+import binascii
+import os
+import sys
+
+## print header first 
+## print header first
+authorSignature = '\nedeterminer.py - Published by Laura Tzigiannis and Moritz Nentwig\n'
+authorSignature += '---------------------------------------------------------------\n'
+print(authorSignature)
+
+## tests
+def check_file_extension(file): 
+    file_extension = file[file.rfind("."):]
+    if file_extension == ".aes":
+        return("AES")
+    elif file_extension == ".des":
+        return("DES")
+    elif file_extension == ".3des":
+        return("3DES")
+    elif file_extension == ".idea":
+        return("IDEA")
+    elif file_extension == ".bf":
+        return("Blowfish")
+    else:
+        return("Unrecognized file extension")
+
+def check_file_header(file):
+    encryption_algorithms = ["AES", "DES", "3DES", "IDEA", "Blowfish"]
+    file_header = file.read(128)
+    for algorithm in encryption_algorithms: 
+        if algorithm.encode('utf-8') in binascii.hexlify(file_header):
+            return algorithm 
+    return("No encryption algorithm found in file header")
+
+def check_block_size(file):
+    cipher_file_length = len(file.read())
```

Comment:
checked and print(cipher_file_length) gives me a 0 every time. Tested it on encryptio files and other files.